modules/analysis/rosbag-image-pipeline/modulestack.yaml

```diff
@@ -25,25 +28,29 @@ Resources:
               - "s3:Get*"
               - "s3:List*"
             Effect: Allow
-            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
+            Resource: 
+              - !Sub 
+                - "arn:aws:s3:::${ProjectName}*"
+                - ProjectName: !Ref ProjectName
+              - "arn:aws:s3:::cdk*"
           - Effect: Allow
             Action:
               - "ecr:Describe*"
               - "ecr:Get*"
               - "ecr:List*"
               - "ecr:BatchGetImage"
             Resource:
-              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/addf-${DeploymentName}-${ModuleName}*"
+              - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/${ProjectName}-${DeploymentName}-${ModuleName}*"
```

Comment:
You have ecr:* below. Do you still need these set of permissions?

Reply:
It's possible its redundant with the the prior permissions, yes. This is from Kevin Soucy's initial commit 2 years ago. Should we test this with removal now or in a separate PR? @dgraeber @srinivasreddych

Reply:
https://github.com/awslabs/autonomous-driving-data-framework/issues/503